bin/log-perormance-results.js

```diff
@@ -39,20 +38,27 @@ const data = new TextEncoder().encode(
 		metrics: resultsFiles.reduce( ( result, { metricsPrefix }, index ) => {
 			return {
 				...result,
-				...mapKeys(
-					performanceResults[ index ][ hash ],
-					( _, key ) => metricsPrefix + key
+				...Object.keys( performanceResults[ index ][ hash ] ).reduce(
+					( accumulator, key ) => {
+						accumulator[ metricsPrefix + key ] =
+							performanceResults[ index ][ hash ][ key ];
+						return accumulator;
+					},
+					{}
```

Comment:
This could be a simpler and more comprehensible way to do it: 

```suggestion
				...Object.fromEntries(
					Object.entries( performanceResults[ index ][ hash ] )
						.map( ( [ key, value ] ) =>[ metricsPrefix + key, value ] )
				),
```

But I don't have strong feelings about it.

Reply:
I like it indeed.